components/authz-service/server/v2/validation.go

```diff
@@ -0,0 +1,75 @@
+package v2
+
+import (
+	"fmt"
+	"regexp"
+
+	constants "github.com/chef/automate/components/authz-service/constants"
+	"github.com/pkg/errors"
+)
+
+var emptyOrWhitespaceOnlyRE = regexp.MustCompile(`^\s*$`)
+
+func confirmRequiredIDandName(id, name, resourceName string) error {
+	err := confirmRequiredID(id, resourceName)
+	if err != nil {
+		return err
+	}
+	err = confirmRequiredName(name, resourceName)
+	return err
+}
+
+func confirmRequiredField(field, fieldName, resourceName string) error {
+	if emptyOrWhitespaceOnlyRE.MatchString(field) {
+		e := fmt.Sprintf("a %s %s is required and must contain at least one non-whitespace character", resourceName, fieldName)
+		return errors.New(e)
```

Comment:
[nit]
```suggestion
		return fmt.Errorf("a %s %s is required and must contain at least one non-whitespace character", resourceName, fieldName)
```

Below, we're using [`errors.Errorf`](https://github.com/chef/automate/pull/3312/files#diff-80445eaad60cf019d0dd4c54dc9a488dR69), that would work, too.